doc/3/controllers/document/get/snippets/get.java

```diff
@@ -0,0 +1,4 @@
+    ConcurrentHashMap<String, Object> response =
+    kuzzle.getDocumentController().get("nyc-open-data", "yellow-taxi", "some-id")
+    .get();
+    System.out.println(response);
```

Comment:
Can you add the result of the `println` in comment? 
It helps developer to better apprehend the structure of the response object they have. We try to do this for every method (Look the [Javascript SDK](https://github.com/kuzzleio/sdk-javascript/blob/master/doc/7/controllers/document/create/snippets/create.js) and [Doc](https://docs.kuzzle.io/sdk/js/7/controllers/document/create/#usage))